### PR TITLE
Assignment 1 with Vera Korchemnaya (@VeraKorchemnaya)

### PR DESCRIPTION
**Pull request by Wvvern**
Issue 1 and 2 resolved
Issue 3: Implemented and unit tested additional feature
- Menu for selecting the difficulty of the game with new easy, medium, and hard questions
Updated to latest version of branch.

### CODE REVIEW

**Comment by VeraKorchemnaya** [comment]
We got our questions from these two sources: 
[funtrivia.com](https://www.funtrivia.com/en/Movies/Princess-Bride-The-877.html)
[quiz-bliss.com](https://www.quiz-bliss.com/alisegel/quiz-can-you-answer-22-questions-every-princess-bride-fan-should-get)

**Comment by GeorgeKoralski007** [comment]
- Pull Request title includes "Assignment 1" in the title ✔
- Pull Request targets Assignment 1 branch ✔
- Issue 1: Application no longer crashes ✔
- Issue 2: Unit test properly passes ✔

### Extra Credit
Come up with a new feature for the application

- Issue 3: Add a new feature request implemented ✔
- Issue 3: Feature request unit tested ✔

Improvement suggestions:
When displaying the result percentage, round to 2 decimal points.

File: PrincessBrideTrivia/PrincessBrideTrivia/Program.cs [hunk 1]
```diff
@@ -24,7 +24,7 @@ public static void Main(string[] args)
 
         public static string GetPercentCorrect(int numberCorrectAnswers, int numberOfQuestions)
         {
-            return (numberCorrectAnswers / numberOfQuestions * 100) + "%";
+            return ((double)numberCorrectAnswers / numberOfQuestions * 100) + "%";
```
Comment (Howard404) [inline PrincessBrideTrivia/PrincessBrideTrivia/Program.cs:31]: Casting to a double clears the unit test. I would suggest casting the operation into a variable type double and return the variable as an integer. No decimals will make the result look clean. (Optional)

**Review by Howard404 — COMMENTED** [review]
Pull Request title includes "Assignment 1" in the title ✔
Pull Request targets Assignment 1 branch ✔
Issue 1: Application no longer crashes ✔
Issue 2: Unit test properly passes ✔

**Comment by JLee35** [comment]
Great job! I loved the difficulty feature.

**Pull request closed without merging**